tests/e2e/array-info.sh

```diff
@@ -73,6 +73,7 @@ export PMAX_TENANT_PREFIX="tn1"
 export PSTORE_USER="username"
 export PSTORE_PASS="password"
 export PSTORE_GLOBALID="myglobalpstoreid"
+# ip only, do not include /api/rest at end
```

Comment:
In this file, since we're here-- please remove the PScale endpoint's port number:

`export PSCALE_ENDPOINT="1.1.1.1:8080"`
should be 
`export PSCALE_ENDPOINT="1.1.1.1"`

Reply:
updated